repo2docker/buildpacks/conda/environment.py-3.5.yml

```diff
@@ -1,3 +1,5 @@
+# py3.5 is not being auto-generated from this file anymore
+# update environment.py-3.5.frozen.yml directly
```

Comment:
Should we delete all contents from this file except for the comment? So that using this file fails instead of working and doing the wrong thing?

I think we should keep this file as a pointer and place to store this comment so that we don't end up thinking the file got deleted by accident.

Reply:
yeah, good call.